tests/week-2/d-predicates.test.js

```diff
@@ -0,0 +1,33 @@
+const { isNegative, isBelow9 } = require("../../week-2/D-predicates/exercise");
+
+test("'isNegative' is a predicate", () => {
+  expect(typeof isNegative(2)).toBe("boolean");
+});
+
+test("'isBelow9' is a predicate", () => {
+  expect(typeof isBelow9(2)).toBe("boolean");
+});
+
+test("'isNegative' returns true when number is negative", () => {
```

Comment:
I guess this is testing both negative and positive numbers, and not just negative - should we update the test text?